cli/azd/test/functional/telemetry_test.go

```diff
@@ -299,7 +302,7 @@ func Test_CLI_Telemetry_NestedCommands(t *testing.T) {
 			require.Contains(t, m, fields.CmdEntry)
 			require.Equal(t, "cmd.up", m[fields.CmdEntry])
 
-			require.NotContains(t, m, fields.CmdFlags)
+			//require.NotContains(t, m, fields.CmdFlags)
```

Comment:
Should we be concerned about this?  I don't really grok what this was asserting.  Any ideas?